CONTRIBUTING.md

```diff
@@ -19,7 +19,11 @@ your change directly to the repository:
 
 # Coding Conventions
 
+<<<<<<< HEAD
```

Comment:
Remove the lines beginning with `<<<<` through `====`.  This is where a conflict in the git merge occurred, and only one of the sentences needs to remain.  You also need to remove the line beginning `>>>>`.